redis.conf

```diff
@@ -1697,6 +1697,30 @@ slowlog-max-len 128
 # "CONFIG SET latency-monitor-threshold <milliseconds>" if needed.
 latency-monitor-threshold 0
 
+################################ LATENCY STATS ##############################
+
+# The Redis extended latency stats track per command latencies and enable
+# exporting the percentile distribution and cumulative latency distributions,
+# via the INFO LATENCYSTATS command.
+# We have two groups of outputs on INFO LATENCYSTATS:
+#   - latency by percentile distribution.
+#   - cumulative distribution of latencies.
+# Using the cumulative distribution of latencies you can merge several stats
+# from different nodes to calculate aggregate metrics. Note that percentile
+# distribution is not mergeable between nodes.
+#
+# By default, the extended latency monitoring is enabled since the overhead
+# of keeping track of the command latency is very small.
+# if you don't want to track extended latency metrics, you can easily disable it
+# at runtime using the command:
+# "CONFIG SET latency-track no" if needed.
+# latency-track yes
```

Comment:
We might instead omit this additional boolean flag entirely and just use`latency-track-percentile`. If latency-track-percentile is set to an empty string, it disables latency tracking.

Reply:
that's what i was arguing too.. similar to `save` config.

Reply:
@oranagra and @madolson there might be users that are not interested in the percentiles but want to keep track of latency for the histograms exporting and merging. 
Meaning they don't want "latency by percentile distribution"  but only the "cumulative distribution of latencies". Can we keep the config to enable disable this latency tracking and the config for percentiles? Do you agree?

Reply:
resolving giving the discussion on https://github.com/redis/redis/pull/9462#discussion_r776388159